docs/ref/middleware.txt

```diff
@@ -464,6 +466,69 @@ in the :setting:`SECURE_REDIRECT_EXEMPT` setting.
 
 .. _nginx: https://nginx.org/
 
+.. _content-security-policy:
+
+Content Security Policy
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think all CSP information should be a separate howto page, like the clickjacking and csrf topics.